dockernet/config.sh

```diff
@@ -241,6 +255,36 @@ EVMOS_MAIN_CMD="$EVMOS_BINARY --home $DOCKERNET_HOME/state/${EVMOS_NODE_PREFIX}1
 EVMOS_RECEIVER_ADDRESS='evmos123z469cfejeusvk87ufrs5520wmdxmmlc7qzuw'
 EVMOS_MICRO_DENOM_UNITS="000000000000000000000000"
 
+# DYDX
+DYDX_CHAIN_ID=DYDX
+DYDX_NODE_PREFIX=dydx
+DYDX_NUM_NODES=1
+DYDX_BINARY="$DOCKERNET_HOME/../build/dydxprotocold"
+DYDX_VAL_PREFIX=val
+DYDX_ADDRESS_PREFIX=dydx
+DYDX_REV_ACCT=rev
```

Comment:
good eye! But this was intentional

Accounts for different chains are independent so there's no reason we can't have a `val` on each chain. Eventually, I'm going to remove all the instances of `{n}val` so that we don't have to use a variable to loop up the name for each chain. But that's for another PR...